_posts/2023-01-04-trino-2022-the-rabbit-reflects.md

```diff
@@ -0,0 +1,581 @@
+
+Martin, Manfred, and Matt released the [second version of the Trino Definitive
+Guide]({% post_url 2022-10-03-the-definitive-guide-2 %}). This 
+version fixed errata, added the deployment process to include newer Kubernetes
+installation methods, and updated features with the latest version of Trino that
+was not available in the initial version of the book. Along with this, there are
+[efforts underway to translate this book](https://simpligility.ca/2022/12/trino-guide-for-everyone-in-2023/)
+to different languages. Huge thanks to everyone involved in this effort!
+
+### Starburst provides Trino in the cloud
+
+As a major community supporter, [Starburst](https://starburst.io/) helped us
+with events, marketing, developer relations, and partner cooperation. Starburst
+also provided a large part of development and code contributions to Trino and
+its related projects. Starburst acquired Varada and integrated the object
+storage indexing technology, and they shipped many Starburst Enterprise releases
+for self-managed deployments. On top of all that amazing work, Starburst
+launched [Starburst Galaxy](https://www.starburst.io/platform/starburst-galaxy/)
+as a powerful, muli-cloud SaaS offering of Trino. Security, cluster management,
+a query editor, and many other features are included in this new platform.
+
+### Amazon upgrades Athena
+
+[Athena version three rolled out]({% post_url 2022-12-01-athena %})
+and is now based on a recent Trino release. This is great news for Athena users
+who were missing the many performance gains, expanded SQL support, and other
+features from Trino, since the prior versions are based on old Presto releases.
+As a result, the large Athena community and their feedback and knowledge have
+become more integrated with the Trino community, and we are seeing positive
+impact for Trino releases already.
+
+### dbt-trino
+
+dbt users rejoice! The [official dbt Trino integration
+](https://docs.getdbt.com/reference/warehouse-setups/trino-setup) made it into
+dbt this year! This means that anyone who wanted to read or write data from or
+to multiple data sources is now able to. If you want to dive into all you need
+to know to get started and a tutorial, [check out this blog post
+](https://docs.starburst.io/blog/2022-11-30-dbt0-introduction.html) written by
+the contributors of this integration.
+
+### Python client improvements
+
+The trajectory of the [trino-python-client
+](https://github.com/trinodb/trino-python-client) doubled this year. There were
+a lot of performance improvements around the integration with the sqlalchemy
+integration. This also included a range of bug fixes.
+
+### Airflow integration
+
+The long-awaited [Trino/Airflow integration
+](https://airflow.apache.org/docs/apache-airflow-providers-trino/stable/index.html)
+landed this year. This paired well with the new task-retry and fault-tolerant
+execution features. There was a small tutorial on [how to use Trino and Airflow
```

Comment:
We should probably not link to this links since it does NOT use the proper support that is now available.

Reply:
Ah, fair